dataflow/src/org/checkerframework/dataflow/analysis/Store.java

```diff
@@ -54,6 +54,26 @@
     S leastUpperBound(S other);
 
     /**
+     * Compute an upper bound of two stores that is wider than the least upper bound of the two
+     * stores.
+     *
+     * <p>(This might not be require for an analysis, if not required, implementations should call
```

Comment:
Would this sentence be clearer as 'A particular analysis might not require widening and will instead use leastUpperBound.`
Currently the sentence seems to suggest that an analysis has to implement something specific, where it actually should say that if you don't need this feature, don't call it. But maybe I'm misunderstanding what it is trying to say...